core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -415,6 +415,10 @@
   TSERV_COMPACTION_SERVICE_ROOT_PLANNER("tserver.compaction.major.service.root.planner",
       DefaultCompactionPlanner.class.getName(), PropertyType.CLASSNAME,
       "Compaction planner for root tablet service"),
+  TSERV_COMPACTION_SERVICE_ROOT_THROUGHPUT("tserver.compaction.major.service.root.throughput", "0B",
+      PropertyType.BYTES,
+      "Maximum number of bytes to read or write per second over all major"
+          + " compactions in this compaction service, or 0B for unlimited."),
```

Comment:
I think a property that is easier to spell is more user friendly.  The last part of the property could be "limit" or "rate.limit".  As a bad speller, I know a word like "throughput" would trip me up.

Reply:
I like rate.limit better.  Changed to rate.limit in abc961a